clients/v1_auth/v1_auth.go

```diff
@@ -0,0 +1,428 @@
+	for _, a := range auths.GetAuthorizations() {
+		var permissions []string
+		for _, p := range a.GetPermissions() {
+			permissions = append(permissions, permString(p))
+		}
+
+		usr, err := c.UsersApi.GetUsersID(ctx, a.GetUserID()).Execute()
+		if err != nil {
+			return fmt.Errorf("could not find user with ID %q: %w", a.GetUserID(), err)
+		}
+
+		tokens = append(tokens, v1Token{
+			ID:          a.GetId(),
+			Description: a.GetDescription(),
+			Token:       a.GetToken(),
+			Status:      a.GetStatus(),
+			UserName:    usr.GetName(),
+			UserID:      usr.GetId(),
+			Permissions: permissions,
+		})
+	}
+
+	return c.printV1Tokens(&v1PrintOpts{tokens: tokens})
+}
+
+type ActiveParams struct {
+	clients.AuthLookupParams
+}
+
+func (c Client) SetActive(ctx context.Context, params *ActiveParams, active bool) error {
+	id, err := c.getAuthReqID(ctx, &params.AuthLookupParams)
+	if err != nil {
+		return err
+	}
+
+	req := c.AuthorizationsApi.PatchAuthorizationsID(ctx, id)
+	var status string
+	if active {
+		status = "active"
+	} else {
+		status = "inactive"
+	}
+	req.AuthorizationUpdateRequest(api.AuthorizationUpdateRequest{
+		Status: &status,
+	})
+
+	auth, err := req.Execute()
+	if err != nil {
+		return fmt.Errorf("could not find Authorization with ID %q: %w", id, err)
+	}
+
+	usr, err := c.UsersApi.GetUsersID(ctx, auth.GetUserID()).Execute()
+	if err != nil {
+		return fmt.Errorf("could not find user from User ID %q: %w", auth.GetUserID(), err)
+	}
+
+	ps := make([]string, 0, len(auth.GetPermissions()))
+	for _, p := range auth.GetPermissions() {
+		ps = append(ps, permString(p))
+	}
+
+	return c.printV1Tokens(&v1PrintOpts{
+		token: &v1Token{
+			ID:          auth.GetId(),
+			Description: auth.GetDescription(),
+			Token:       auth.GetToken(),
+			Status:      auth.GetStatus(),
+			UserName:    usr.GetName(),
+			UserID:      usr.GetId(),
+			Permissions: ps,
+		},
+	})
+}
+
+type SetPasswordParams struct {
+	clients.AuthLookupParams
+	Password string
+}
+
+func (c Client) SetPassword(ctx context.Context, params *SetPasswordParams) error {
+	id, err := c.getAuthReqID(ctx, &params.AuthLookupParams)
+	if err != nil {
+		return err
+	}
+
+	auth, err := c.AuthorizationsApi.GetAuthorizationsID(ctx, id).Execute()
+	if err != nil {
+		return fmt.Errorf("could not find authorization with User ID %q: %w", id, err)
+	}
+
+	password := params.Password
+	if password == "" {
+		password, err = c.StdIO.GetSecret("Please enter new password", 0)
+		if err != nil {
+			return err
+		}
+	}
+
+	if err := c.UsersApi.PostUsersIDPassword(ctx, auth.GetUserID()).Execute(); err != nil {
+		return fmt.Errorf("error setting password: %w", err)
+	}
+
+	return nil
+}
+
+func (c Client) printV1Tokens(params *v1PrintOpts) error {
+	if c.PrintAsJSON {
+		var v interface{}
+		if params.token != nil {
+			v = params.token
+		} else {
+			v = params.tokens
+		}
+		return c.PrintJSON(v)
+	}
+
+	headers := []string{
+		"ID",
+		"Description",
+		"Token",
```

Comment:
```suggestion
		"Name / Token",
```
For 1:1 with old CLI